pkg/store/storepb/custom.go

```diff
@@ -484,6 +484,7 @@ type SeriesStatsCounter struct {
 	Series  int
 	Chunks  int
 	Samples int
+	Bytes   int
```

Comment:
Use uint64 since int overflows with a value > 2GiB which is more likely in Pantheon. One million time series fetched may reach 2GiB data fetched.